components/Dialogs/examples/DialogsCustomShadowExampleViewController.swift

```diff
@@ -42,12 +42,23 @@ class CustomShadowViewController: UIViewController {
     bodyLabel.sizeToFit()
     self.view.addSubview(bodyLabel)
 
+    self.view.isAccessibilityElement = true
+    self.view.accessibilityTraits = .button
```

Comment:
Shouldn't the view be a container element?

Reply:
I don't believe so; the view itself is acting as the button here due to the tap gesture.